docs/src/man/seq.md

```diff
@@ -99,7 +99,7 @@ ambiguous nucleotide can take. For example, `DNA_R` (meaning the nucleotide is
 either `DNA_A` or `DNA_G`) is encoded as `0101` because `0101` is the bitwise OR
 of `0001` (`DNA_A`) and `0100` (`DNA_G`). The gap symbol is always `0000`.
 
-|    Nucleotide    |  Bits  |
+|    NucleicAcid    |  Bits  |
```

Comment:
Extreme pedantry here: Is it worth bothering to line `|` chars up in these tables? If this is annoying in your editor, I can do it later using Tabular.vim